deployment/default.nix

```diff
@@ -112,5 +114,6 @@ let
     david = mkEnv "david" "eu-west-1";
     alpha = mkEnv "alpha" "eu-west-2";
     pablo = mkEnv "pablo" "eu-west-3";
+    wyohack = mkEnv "wyohack" "eu-central-1";
```

Comment:
Maybe it would make sense to put it into a US server instead :)
No big deal anyway

Reply:
good point except it's already done now and it will be a PITA to change it. Hmm, maybe I should though.

Reply:
phew, that took some time